frontend/components/user/settings/UserSettingsPassword.vue

```diff
@@ -0,0 +1,147 @@
+<script lang="ts" setup>
+import { object as yupObject } from 'yup'
+import { useForm } from 'vee-validate'
+import { API_PATH } from '~/types/customFetch'
+
+const { t: $t } = useI18n()
+const { fetch } = useCustomFetch()
+const toast = useBcToast()
+
+const { handleSubmit, errors, defineField } = useForm({
+  validationSchema: yupObject({
+    oldPassword: passwordValidation($t),
+    newPassword: passwordValidation($t),
```

Comment:
Please add a Validator for the new password that it does not match the ole one

Reply:
I like the idea, done.